src/schema/v2/me/index.ts

```diff
@@ -300,6 +301,7 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
       description: "Collector's position with relevant institutions",
       resolve: collectorProfileResolver("other_relevant_positions"),
     },
+    quiz,
```

Comment:
Sorry for always jumping in here with qs 😅 

Can we remove this from `user`? User is an annoying field and me generally does what we want. No need to keep it in two places unless we're treating ArtQuiz like a dynamic API (`user(id: "foo")`)

Reply:
It's needed for Braze

Reply:
Cool yep

Reply:
> User is an annoying field

lol what does this mean? `User` is a pretty fundamental first-class thing to have in our schema (esp for admin abilities), distinct from `me` (where industry convention sort of landed on that as the currently logged-in user).

or do you mean that the _types_ of `User` and `Me` should be the same (but different resolvers)? that's interesting but I think hard to do in practice and separating 'current user' from all other 'user' stuff is where we landed.

Reply:
Oh I just mean to work with! Like, in order to develop against it you need to set a special X-User-ID header in GraphiQL which is easy to forget, etc. `user` seems more generalizable than `me`, which is personal. That's all 😄 And nope, don't think that they should be the same. Just that one is more annoying to work with in practice